targets/x86/profiles/x64/config

```diff
@@ -431,7 +160,6 @@ CONFIG_BUILD_PATENTED=y
 # CONFIG_BUILD_NLS is not set
 CONFIG_SHADOW_PASSWORDS=y
 # CONFIG_CLEAN_IPKG is not set
-CONFIG_CLEAN_IPKG_PARTIAL=y
```

Comment:
This should not be removed

Reply:
> > Also, can you please provide details of why we need the two patches? These look very hard to maintain between kernel versions. There must be a better way to implement the changes within them if they truly are necessary. Otherwise this should perhaps remain as an out of tree change.

Some changes in the kernel configuration are either the CONFIG_KERNEL_ in the openwrt root directory .config I submitted earlier, or this time openwrt file target/linux/x86/generic/config-default patch.
This is the linux .config file generation process.
/home/fengchen/openwrt/gargoyle-18.06/x86-src/scripts/kconfig.pl  + + /home/fengchen/openwrt/gargoyle-18.06/x86-src/target/linux/generic/config-4.14 /home/fengchen/openwrt/gargoyle-18.06/x86-src/target/linux/x86/config-4.14 /home/fengchen/openwrt/gargoyle-18.06/x86-src/target/linux/x86/64/config-default > /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.target
awk '/^(#[[:space:]]+)?CONFIG_KERNEL/{sub("CONFIG_KERNEL_","CONFIG_");print}' /home/fengchen/openwrt/gargoyle-18.06/x86-src/.config >> /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.target
echo "# CONFIG_KALLSYMS_EXTRA_PASS is not set" >> /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.target
echo "# CONFIG_KALLSYMS_ALL is not set" >> /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.target
echo "CONFIG_KALLSYMS_UNCOMPRESSED=y" >> /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.target
/home/fengchen/openwrt/gargoyle-18.06/x86-src/scripts/package-metadata.pl kconfig /home/fengchen/openwrt/gargoyle-18.06/x86-src/tmp/.packageinfo /home/fengchen/openwrt/gargoyle-18.06/x86-src/.config 4.14 > /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.override
/home/fengchen/openwrt/gargoyle-18.06/x86-src/scripts/kconfig.pl 'm+' '+' /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.target /dev/null /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.override > /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set
mv /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.old
grep -v INITRAMFS /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.old > /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set
echo 'CONFIG_INITRAMFS_SOURCE=""' >> /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set
echo '# CONFIG_INITRAMFS_FORCE is not set' >> /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set
rm -rf /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/modules
cmp -s /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.prev || { cp /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config; cp /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.set /home/fengchen/openwrt/gargoyle-18.06/x86-src/build_dir/target-x86_64_musl/linux-x86_64/linux-4.14.68/.config.prev; }